router/core/retry_expression_builder.go

```diff
@@ -0,0 +1,85 @@
+package core
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/wundergraph/cosmo/router/internal/expr"
+	"github.com/wundergraph/cosmo/router/internal/retrytransport"
+	"go.uber.org/zap"
+)
+
+const DefaultRetryExpression = "IsRetryableStatusCode() || IsConnectionError() || IsTimeout()"
```

Comment:
I think this doesn't have to be exported, or could be moved into the package `retrytransport`

Reply:
Done